source/monitoring-logging/logging.txt

```diff
@@ -0,0 +1,195 @@
+
+.. meta::
+   :keywords: debugging, printing
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+library-short+} to
+set up and configure **logging**. Logging allows you to receive
+information about database operations, server connections, errors,
+and other events that occur while your application runs.
+
+The {+library-short+} supports `Psr\\Log\\LoggerInterface
+<https://www.php-fig.org/psr/psr-3/#3-psrlogloggerinterface>`__, a PSR-3
+logger interface that configures your application to receive log messages.
+You can register one or more ``Psr\Log\LoggerInterface`` objects to
+send messages to the logger. The {+library-short+} is built on top of
+``libmongoc`` and the {+extension-short+}, so the logger receives event notifications
+from both components.
+  
+Register a Logger
+-----------------
+
+To configure your application to receive messages about driver events,
+create a logger class that implements the ``Psr\Log\LoggerInterface``
+interface. Then, use the ``MongoDB\add_logger()`` function to register
+your logger.
+
+Each log message includes the following information:
+
+- Log level: Indicates the severity of the message. To view a list of 
+  possible levels, see `PSR\\Log\\LogLevel <https://www.php-fig.org/psr/psr-3/#5-psrlogloglevel>`__.
+- Message: Describes the logged event.
+- Domain string: Specifies the driver component that emitted the log message.
+
+To implement ``Psr\Log\LoggerInterface``, you can create a class that
+extends the ``Psr\Log\AbstractLogger`` class. ``AbstractLogger`` implements
+``LoggerInterface`` and a generic ``log()`` method, which receives log
+messages at each severity level.
+
+Example
+~~~~~~~
+
+This example performs the following actions:
+
+- Creates a logger called ``MyLogger`` that extends the ``AbstractLogger`` class
+  and records the log level, description, and domain of each event
+- Registers the logger
+- Creates a client that connects to a CosmosDB cluster
+- Prints each log message
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/monitoring-logging/logging.php
+      :start-after: start-register-logger
+      :end-before: end-register-logger
+      :language: php
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      Array
+      (
+          [0] => Array
+              (
+                  [0] => debug
+                  [1] => Connection string: 'mongodb://a.mongo.cosmos.azure.com:19555/'
+                  [2] => PHONGO
+              )
+          [1] => Array
+              (
+                  [0] => debug
+                  [1] => Creating Manager, phongo-1.21.0[stable] - mongoc-1.30.1(bundled), libbson-1.30.1(bundled), php-8.4.4
+                  [2] => PHONGO
+              )
+          [2] => Array
+              (
+                  [0] => debug
+                  [1] => Setting driver handshake data: { name: 'ext-mongodb:PHP / PHPLIB ', version: '1.21.0 / 1.21.1 ', platform: 'PHP 8.4.4 ' }
+                  [2] => PHONGO
```

Comment:
Q: What is PHONGO? might be worth describing what types of things users might expect to see here

Reply:
It means the log entry comes from the PHP extension. I will ask dbx if PHONGO (the extension) and mongoc (the c driver) are the only possible domains and I'll add that information